client/components/scheduled-updates/scheduled-updates-gate/upsell-nudge.tsx

```diff
@@ -0,0 +1,36 @@
+import { PLAN_BUSINESS, getPlan } from '@automattic/calypso-products';
+import { addQueryArgs } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import UpsellNudge from 'calypso/blocks/upsell-nudge';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+const UpsellNudgeNotice = () => {
```

Comment:
Do we have a component somewhere else that does this already? Is it the redirect that makes it necessary?

Reply:
> Do we have a component somewhere else that does this already?

Not really, the strings are all different in other cases.

> Is it the redirect that makes it necessary?

The intention here is to keep `ScheduledUpdatesGate` as clean as possible. Also, it touches on a few props and I thought moving it into it's own component makes it cleaner.